insights-notification-schemas-java/src/main/resources/schemas/Action.json

```diff
@@ -34,10 +34,7 @@
       "description": "ISO-8601 formatted date (per platform convention) when the message was sent in UTC. Dates with timezones/offsets are rejected.",
       "title": "Sent timestamp"
     },
-    "account_id": {
-      "type": "string",
-      "title": "Account id"
-    },
+    "account_id": true,
```

Comment:
This tells the json-schema that we are having an `account_id` property here, anything is fine. We will refine it below